src/Uno.Extensions.Templates/Uno.Extensions.Templates.csproj

```diff
@@ -19,55 +19,75 @@
 		<PackageProjectUrl>https://github.com/unoplatform/uno.extensions</PackageProjectUrl>
 		<PackageIconUrl>https://nv-assets.azurewebsites.net/logos/uno.png</PackageIconUrl>
 		<RepositoryUrl>https://github.com/unoplatform/uno.extensions</RepositoryUrl>
-		<Description></Description>
-		<Copyright>Copyright (C) 2015-2022 nventive inc. - all rights reserved</Copyright>
+		<Description>Quickstart template for using Uno Extensions.</Description>
 		<GeneratePackageOnBuild Condition="'$(Configuration)'=='Release'">true</GeneratePackageOnBuild>
 		<PackageTags>dotnet-new;templates;uno-platform;webassembly;ios;android;winui;macos;uno-extensions</PackageTags>
 		<NoWarn>$(NoWarn);NU5128</NoWarn>
 	</PropertyGroup>
 
 	<ItemGroup>
 		<UpToDateCheckInput Include="content\**" />
+		<Compile Remove="content/**/*" />
+		<Page Remove="content/**/*" />
+		<Content Remove="content/**/*" />
+		<Content Include="content/**/*"
+				Exclude="**/obj/**;**/bin/**;**/*.Debugging.sln"
+				KeepDuplicates="False"
+				PackagePath="content"
+				Pack="true" />
+		<Content Include="content/unoapp-extensions/MyExtensionsApp.Windows/app.manifest"
```

Comment:
Are these individual Content includes required?

Reply:
Yes they are required. If you look at the package path it duplicates them from the Windows to the Linux / GTK heads